File: Coding/coding modul 4/PRAK402-2410817110007-MUHAMMAD-HAMKA-AKBARI.c
#include <stdio.h>

void printOddNumbers(int max) {
    for (int i = 1; i <= max; i++) {
        if (i % 2 != 0) {  // Check if the number is odd
            printf("%d ", i);
        }
    }
    printf("\n");
}

void printEvenNumbersReversed(int max) {
    for (int i = max; i >= 2; i--) {
        if (i % 2 == 0) {  // Check if the number is even
            printf("%d ", i);
        }
    }
    printf("\n");
}

int main() {
    int max;
    
    // Input the maximum number
    printf("Enter the maximum number: ");
    scanf("%d", &max);

    // Print odd numbers from 1 to max
    printOddNumbers(max);

    // Print even numbers from max to 2 in reverse order
    printEvenNumbersReversed(max);

    return 0;
}

File: Coding/coding modul 4/PRAK402-2410817110007-MUHAMMAD-HAMKA-AKBARI.py
def print_odd_numbers(max_num):
    # Print odd numbers from 1 to max_num
    for i in range(1, max_num + 1):
        if i % 2 != 0:
            print(i, end=" ")
    print()

def print_even_numbers_reversed(max_num):
    # Print even numbers from max_num to 2 in reverse
    for i in range(max_num, 1, -1):
        if i % 2 == 0:
            print(i, end=" ")
    print()

# Main program
max_num = int(input("Enter the maximum number: "))

# Call the functions to print results
print_odd_numbers(max_num)
print_even_numbers_reversed(max_num)
